sdk/search/search/review/search.api.md

```diff
@@ -10,6 +10,48 @@ import { PipelineOptions } from '@azure/core-http';
 import { ServiceClientCredentials } from '@azure/core-http';
 import { WebResource } from '@azure/core-http';
 
+// @public
+export interface AccessCondition {
+    ifMatch?: string;
+    ifNoneMatch?: string;
+}
+
+// @public
+export interface Analyzer {
+    name: string;
+    odatatype: "Analyzer";
+}
+
+// @public
+export interface AnalyzeRequest {
+    analyzer?: AnalyzerName;
+    charFilters?: CharFilterName[];
+    text: string;
+    tokenFilters?: TokenFilterName[];
+    tokenizer?: TokenizerName;
+}
+
+// @public
+export interface AnalyzeResult {
+    tokens: TokenInfo[];
+}
+
+// @public
+export type AnalyzerName = 'ar.microsoft' | 'ar.lucene' | 'hy.lucene' | 'bn.microsoft' | 'eu.lucene' | 'bg.microsoft' | 'bg.lucene' | 'ca.microsoft' | 'ca.lucene' | 'zh-Hans.microsoft' | 'zh-Hans.lucene' | 'zh-Hant.microsoft' | 'zh-Hant.lucene' | 'hr.microsoft' | 'cs.microsoft' | 'cs.lucene' | 'da.microsoft' | 'da.lucene' | 'nl.microsoft' | 'nl.lucene' | 'en.microsoft' | 'en.lucene' | 'et.microsoft' | 'fi.microsoft' | 'fi.lucene' | 'fr.microsoft' | 'fr.lucene' | 'gl.lucene' | 'de.microsoft' | 'de.lucene' | 'el.microsoft' | 'el.lucene' | 'gu.microsoft' | 'he.microsoft' | 'hi.microsoft' | 'hi.lucene' | 'hu.microsoft' | 'hu.lucene' | 'is.microsoft' | 'id.microsoft' | 'id.lucene' | 'ga.lucene' | 'it.microsoft' | 'it.lucene' | 'ja.microsoft' | 'ja.lucene' | 'kn.microsoft' | 'ko.microsoft' | 'ko.lucene' | 'lv.microsoft' | 'lv.lucene' | 'lt.microsoft' | 'ml.microsoft' | 'ms.microsoft' | 'mr.microsoft' | 'nb.microsoft' | 'no.lucene' | 'fa.lucene' | 'pl.microsoft' | 'pl.lucene' | 'pt-BR.microsoft' | 'pt-BR.lucene' | 'pt-PT.microsoft' | 'pt-PT.lucene' | 'pa.microsoft' | 'ro.microsoft' | 'ro.lucene' | 'ru.microsoft' | 'ru.lucene' | 'sr-cyrillic.microsoft' | 'sr-latin.microsoft' | 'sk.microsoft' | 'sl.microsoft' | 'es.microsoft' | 'es.lucene' | 'sv.microsoft' | 'sv.lucene' | 'ta.microsoft' | 'te.microsoft' | 'th.microsoft' | 'th.lucene' | 'tr.microsoft' | 'tr.lucene' | 'uk.microsoft' | 'ur.microsoft' | 'vi.microsoft' | 'standard.lucene' | 'standardasciifolding.lucene' | 'keyword' | 'pattern' | 'simple' | 'stop' | 'whitespace';
+
+// @public
+export type AnalyzerUnion = Analyzer | CustomAnalyzer | PatternAnalyzer | StandardAnalyzer | StopAnalyzer;
```

Comment:
Yeah, it seems like maybe a better way to type this would be

```ts
export type Analyzer = CustomAnalyzer | PatternAnalyzer | StandardAnalyzer | StopAnalyzer;
```

Not sure if there is some easy way to tweak the swagger so it would codegen like this instead

Reply:
The Swagger is fine; The codegen should be tweaked. 😁

Reply:
yes, to be clear, we need to fix the codegen, but we're in the middle of a rewrite that makes it harder to fix the old generator... I hope to move to the new one soon (tm)

Reply:
updated